scripts/build-image.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env sh
+
+# Where's my apt?
+APTGET="$( which apt-get )"
+
+# I'm not here, i won't interact with you
+APTGETCMD="env DEBIAN_FRONTEND=noninteractive ${APTGET}"
+
+# Let's tell apt to not install recommendations,
+# assume always a positive answer, and allow unauthenticated repos
+# Dpkg: you need to install configurations always. Ah, and be quiet.
+APTGETOPTS="-qq \
+            -o Apt::Install-Recommends=false \
+            -o Apt::Get::Assume-Yes=true \
+            -o Apt::Get::AllowUnauthenticated=true \
+            -o DPkg::Options::=--force-confmiss \
+            -o DPkg::Options::=--force-confnew \
+            -o DPkg::Options::=--force-overwrite \
+            -o DPkg::Options::=--force-unsafe-io"
+
+# List of software we need
+DPKGDEPENDS="bzr \
+             curl \
+             git \
+             htop \
+             locate \
+             lsof \
+             multitail \
+             openssl \
+             openssh-client \
+             postgresql-client \
+             postgresql-common \
+             python \
+             python-psycopg2 \
+             python-setuptools \
+             python-pip \
+             python-openssl \
+             python-ndg-httpsclient \
+             python-pyasn1 \
+             python-github \
+             python-click \
+             supervisor \
+             tmux \
+             unzip \
+             vim \
+             vim-nox \
+             w3m \
+             wget"
+
+# Blazing fast dpkg
+{
+	echo 'force-unsafe-io'
+} | tee etc/dpkg/dpkg.cfg.d/speedup > /dev/null
+
+# Don't give me translations man, i can tok inglis
+{
+	echo 'Acquire::Languages "none";'
+} | tee etc/apt/apt.conf.d/no-languages > /dev/null
+
+# This will setup our default locale.
+# Setting these three variables will ensure we have a proper locale environment
+# avoiding the use of PYTHONIOENCODING (See http://stackoverflow.com/a/34378962)
+locale-gen en_US.UTF-8
+update-locale LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8 LC_ALL=en_US.UTF-8
+export LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8 LC_ALL=en_US.UTF-8
+
+# Setting this according to vauxoo/docker-ubuntu-base#4
+export TERM="xterm"
+
```

Comment:
On Thu, Apr 14, 2016 at 7:11 PM, moylop260 <notifications@github.com> wrote:

> We need this value persistent.
> Is it?
>
> If you run
> docker run --rm vauxoo/docker-ubuntu-base printenv | grep xterm
> What is the output?
>

This topic will helps you :

http://stackoverflow.com/questions/1641477/how-to-set-environment-variable-for-everyone-under-my-linux-system

Specifically:

man 8 pam_env <http://linux.die.net/man/8/pam_env>

man 5 pam_env.conf <http://linux.die.net/man/5/pam_env.conf>

If all login services use PAM <http://www.kernel.org/pub/linux/libs/pam/>,
and all login services have session required pam_env.so in their respective
/etc/pam.d/* configuration files, then all login sessions will have some
environment variables set as specified in pam_env's configuration file.

On most modern Linux distributions, this is all there by default -- just
add your desired global environment variables to /etc/security/pam_env.conf.

This works regardless of the user's shell, and works for graphical logins
too (if xdm/kdm/gdm/entrance/… is set up like this).


-- 
Nhomar Hernandez
CEO Vauxoo.
Site: http://vauxoo.com
Twitter: @nhomar
Blog: http://nhomar.com
Github User: https://github.com/nhomar
Odoo Gold Partner
Skype: nhomar00 (Envia mail previo no lo superviso siempre).
HangOut: nhomar@vauxoo.com
Móvil Venezuela:
+58 4144110269
Móvil México:
+52 1 4773933942